blocks/related-resources/related-resources.css

```diff
@@ -92,10 +92,23 @@
     background-position: 50%;
 }
 
-.related-resources a.document-link::after {
+.related-resources .dc-external-type a.document-link::after,
+.related-resources .dc-internal-type a.document-link::after {
     background: url('../../icons/right-arrow-purple-large.svg') no-repeat center center;
 }
 
+.related-resources .dc-video-type a.document-link::after {
+    background: url('../../icons/play-button-purple.svg') no-repeat 20px center;
```

Comment:
We shouldn't have to add purple variations of icons. It is pretty easy in CSS to change the stroke colour of an SVG when it isn't being used as a background colour.

If you add your icons as `<span class="icon icon-play-button"/>` you should be able to leverage the global button deocoration.